SuperRecyclerView/build.gradle

```diff
@@ -20,6 +20,11 @@ repositories {
     jcenter()
 }
 dependencies {
-    compile 'com.android.support:recyclerview-v7:21.0.3'
-    compile 'com.nineoldandroids:library:2.4.0'
-}
\ No newline at end of file
+    def googleSupportLibVer = "24.2.1"
+
+    compile "com.android.support:recyclerview-v7:${googleSupportLibVer}"
+    compile "com.nineoldandroids:library:2.4.0"
+    // FIXME: It's a workaround to have the SuperRecyclerView use the custom
+    // FIXME: RecyclerView internally.
+    compile project(":CustomWidget")
```

Comment:
let's discuss how to implement the separator today. because I not sure it's a good idea to implement it on the RecyclerView.